### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by toshok**
bump all deps (including yarn), and remove uuid since we aren't using it

### CODE REVIEW

**Review by Andarist — APPROVED** [review]

File: .yarnrc.yml
Comment (Andarist) [inline .yarnrc.yml:1]: IIRC it's also advised to commit `.yarn/releases`, like we do in devtools:
https://github.com/replayio/devtools/tree/ec45e5a71fe83f522ded3cb1ac7644640d4013cb/.yarn/releases

Comment (toshok) [reply .yarnrc.yml:1]: huh, TIL.  adding.

Comment (toshok) [reply .yarnrc.yml:1]: is there a command line to create/add that .cjs file?

Comment (Andarist) [reply .yarnrc.yml:1]: I think it's this one: https://yarnpkg.com/cli/set/version

**Pull request merged**